content/ja/handbook/marketing/marketing-operations/pathfactory/content-library.md

```
@@ -0,0 +1,292 @@
+1. 検索を使用して、既に追加されているかどうかを判断します。アセットには異なる URL がある可能性もあるため、徹底的に行ってください。既にライブラリにあるファイルや URL を追加しようとすると、システムから既に存在することが通知されます。たとえば、`https://about.gitlab.com/customers/extra-hop-networks/` と `https://about.gitlab.com/customers/extra-hop-networks` のアップロードは、異なる URL とみなされ、両方ともライブラリにアップロードされます（末尾の `/` に注意）。
+1. アセットの最も価値のあるバージョン（ブログ投稿 vs. ケーススタディまたは PDF）を持っていることを確認してください。
+1. アセットの最新バージョンを持っていることを確認してください。
+1. [ブログスタイルガイド](/handbook/marketing/brand-and-product-marketing/content/editorial-team/) に従ってください。
+
+### コンテンツのアップロード方法
+
+1. コンテンツライブラリの右上にある `Add Content` をクリックします。
+1. アセットをどのようにアップロードするか（Website URL、ファイルアップロード、CSV）を選択します。
+    - Website URL の場合、URL をコピー/貼り付けします。複数の URL の場合、1 行に 1 つずつ貼り付けます。
+    - ファイルアップロードの場合は、このオプションを選択し、ファイルアップロードウィンドウにファイルをドラッグアンドドロップするか、コンピュータから選択します。ファイルは最大 100MB です。
+    - CSV を使用してコンテンツライブラリに複数のアセットをアップロードするには、[一括アップロードの手順に従ってください](#bulk-upload)。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)